js/okex.js

```diff
@@ -2674,7 +2674,7 @@ module.exports = class okex extends Exchange {
         params = this.omit (params, 'type');
         const request = {
             // instType String No Instrument type, MARGIN, SWAP, FUTURES, OPTION
-            'instId': market['id'],
+            'instId': this.safeValue (market, 'id'),
```

Comment:
i could create a separate PR for this or just leave it here

Reply:
@kroitor @frosty00 Resolve this if it's fine to just leave it here

Reply:
we can just use market['id'] here